spacewalk/config/etc/httpd/conf.d/zz-spacewalk-docs.conf

```diff
@@ -0,0 +1,12 @@
+RewriteEngine on
+RewriteOptions inherit
+
+<Directory "/srv/www/htdocs/docs">
+  # Prepend "/product/version/" to the paths, if the original path is not available
+  RewriteCond %{REQUEST_FILENAME} !-f
+  RewriteCond %{REQUEST_FILENAME} !-d
+  # Rewrite only if the prepended path is available
+  RewriteCond %{DOCUMENT_ROOT}/docs/@PRODUCT/@VERSION/$1 -f [OR]
+  RewriteCond %{DOCUMENT_ROOT}/docs/@PRODUCT/@VERSION/$1 -d
```

Comment:
This will not work with release-notes. We need to think about what to do with them.

Reply:
They should work without problem. The first 2 conditions here apply **only** if the original URL is not found. If the URL is available as-is, no rewrite will be made.